docs/inference.rst

```diff
@@ -108,7 +108,8 @@ An example configuration file (named ``inference.ini``) is::
     [static_args]
     ; waveform parameters that will not change in MCMC
     approximant = IMRPhenomPv2
-    f_lower = 19.0
+    f_lower = 18
```

Comment:
Why not set it to 20.0 instead? The waveforms were generated from 20Hz in the O1 PE analysis according to the PE paper. So shouldn't it be nice if the results produced on running this sample script are as close as possible to those that have been reported in the papers?